internal/local/command_kafka_broker_update.go

```diff
@@ -0,0 +1,49 @@
+package local
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/v3/pkg/broker"
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+)
+
+func (c *Command) newKafkaBrokerUpdateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update [id]",
+		Short: "Update local Kafka broker configurations.",
+		Long:  "Update per-broker or cluster-wide Kafka broker configurations.",
+		Args:  cobra.MaximumNArgs(1),
+		RunE:  c.brokerUpdate,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Update configuration values for broker 1.",
+				Code: "confluent kafka broker update 1 --config min.insync.replicas=2,num.partitions=2",
+			},
+			examples.Example{
+				Text: "Update configuration values for all brokers in the cluster.",
+				Code: "confluent kafka broker update --all --config min.insync.replicas=2,num.partitions=2",
+			},
+		),
+	}
+
+	pcmd.AddConfigFlag(cmd)
+	cmd.Flags().Bool("all", false, "Apply configuration update to all brokers in the cluster.")
```

Comment:
I think we agreed to delete this in another comment, and use the `confluent kafka cluster configuration update` commands instead?

Reply:
oops